CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required (VERSION 3.0)
 project (s2n C)
 
+if(POLICY CMP0077)
+    cmake_policy(SET CMP0077 NEW) #option does nothing when a normal variable of the same name exists.
```

Comment:
Please add to the comment taht this is a 3.13 policy so we can easily remove it once the cmake requirements are upgraded some day in the future.

Reply:
https://cmake.org/cmake/help/git-stage/policy/CMP0077.html
> This policy was introduced in CMake version 3.13. CMake version 3.20.20210526-gd693ef8 warns when the policy is not set and uses OLD behavior. Use the cmake_policy() command to set it to OLD or NEW explicitly.

Based on the doc, we need `cmake_policy` for CMake 3.13 and later.